includes/class-plugin.php

```diff
@@ -299,12 +303,7 @@ public function link_to_ocp($medium, $as_html = true)
     {
         $ref = 'oss';
 
-        if (
-            ( defined( 'WP_ROCKET_WEB_MAIN' ) && strpos( (string) WP_ROCKET_WEB_MAIN, 'cloudlinux.com' ) ) ||
-            ( defined( 'WP_ROCKET_UPDATE_PATH' ) && strpos( (string) WP_ROCKET_UPDATE_PATH, 'cloudlinux' ) ) ||
-            ( defined( 'CL_SMART_ADVICE_VERSION' ) && ! empty( CL_SMART_ADVICE_VERSION ) ) ||
-            is_readable( '/opt/alt/php-xray/' )
-        ) {
+        if ( defined( 'WP_REDIS_PATH' ) && strpos( (string) WP_REDIS_PATH, '.clwpos/redis.sock' ) !== false ) {
```

Comment:
@martinkrcho: I've simplified this, any objections?

Reply:
Looks good to me.